src/parameter_input.cpp

```diff
@@ -343,14 +343,14 @@ bool ParameterInput::ParseLine(InputBlock *pib, std::string line, std::string &n
 }
 
 //----------------------------------------------------------------------------------------
-//! \fn void ParameterInput::AddParameter(InputBlock *pb, std::string name,
-//   std::string value, std::string comment)
+//! \fn void ParameterInput::AddParameter(InputBlock *pb, const std::string & name,
+//   std::string value, const std::string & comment)
 //  \brief add name/value/comment tuple to the InputLine singly linked list in block *pb.
 //  If a parameter with the same name already exists, the value and comment strings
 //  are replaced (overwritten).
 
-void ParameterInput::AddParameter(InputBlock *pb, std::string name, std::string value,
-                                  std::string comment) {
+void ParameterInput::AddParameter(InputBlock *pb, const std::string &name,
+                                  std::string value, const std::string &comment) {
```

Comment:
This could be `const std::string &value`, too.
```suggestion
                                  const std::string &value, const std::string &comment) {
```